roles/dev/tasks/main.yml

```diff
@@ -11,12 +11,90 @@
     force: no
 
 - name: install Golang
-  shell: chdir=/usr/local/ creates=/usr/local/go/bin/go tar xfvz /tmp/go1.5.2.linux-amd64.tar.gz
+  shell: tar xfvz /tmp/go1.5.2.linux-amd64.tar.gz
+  args:
+    chdir: /usr/local/
+    creates: /usr/local/go/bin/go
 
 - name: setup golang environment
   copy:
     dest: /etc/profile.d/00golang.sh
     content: "export PATH=/opt/golang/bin:/usr/local/go/bin:$PATH; export GOPATH=/opt/golang"
 
+- name: check packer's version
+  shell: packer --version
+  register: packer_version
+  ignore_errors: yes
+
+- name: download packer
+  get_url:
+    validate_certs: no
+    url: https://releases.hashicorp.com/packer/0.8.6/packer_0.8.6_linux_amd64.zip
+    dest: /tmp/packer_0.8.6_linux_amd64.zip
+    force: no
+  when: packer_version.stdout != "0.8.6"
```

Comment:
just to close, based on slack conversation we agreed that current check should be good to ensure we install a validated version of tool.